tests/agent_equipment/test_installation.py

```
@@ -0,0 +1,252 @@
+            )
+            self.assertEqual(manifest_result.returncode, 0, manifest_result.stderr)
+            installed_manifest = json.loads(manifest_result.stdout)
+            self.assertEqual(
+                installed_manifest["runtime_identity"],
+                "cpython:"
+                f"{sys.version_info.major}.{sys.version_info.minor}."
+                f"{sys.version_info.micro}",
+            )
+            self.assertEqual(tuple(installed_manifest["paths"]), MANIFEST_PATHS)
+
+            managed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)